bundles/org.openhab.binding.hue/src/main/resources/OH-INF/i18n/hue.properties

```diff
@@ -182,3 +273,14 @@ actionDesc = Send a light command with a custom fade time.
 # discovery results
 
 discovery.group.all_lights.label = All lights
+
+# api v2 dynamic actions
+
+dynamics.action.label = Send a dynamic command.
```

Comment:
The usage is to start with a lowercase and without ending with a "." so "send a dynamic command"